shared/js/background/classes/site.es6.js

```diff
@@ -86,7 +86,7 @@ class Site {
         if (this.domain === 'extensions')
             return 'extensions'
 
-        if (this.domain === chrome.runtime.id)
+        if (window.chrome && this.domain === chrome.runtime.id)
             return 'options'
 
```

Comment:
Not specifically related to this change, but the popup looks broken when you've opened e.g. the Options page. The connection string doesn't show, and it says "undefined trackers blocked".

Reply:
Thanks, fixed that elsewhere.